src/frontend/data/queries/fetchList.tsx

```diff
@@ -0,0 +1,29 @@
+import { QueryFunction } from 'react-query';
+
+import { appData } from '../appData';
+
+/**
+ * `react-query` fetcher for lists of items from the Marsha API.
+ */
+export const fetchList: QueryFunction<any> = async ({ queryKey }) => {
```

Comment:
We can not be more specific than `any` ?

Reply:
I'm not familiar with this, can you propose something ? Or maybe @jbpenrath ?

Reply:
`QueryFunction<APIList<any>>` 🤷‍♂️

Reply:
I'm not sure how easy it would be to type this properly. We could make it parametric but it would add even more boilerplate to `data/index.tsx`.

IMO it's not a big deal as it's not propagated any further: hooks we create using fetchList provide correctly typed return values to components.